src/olympia/pages/templates/pages/shield_study_10.html

```diff
@@ -0,0 +1,58 @@
+    <link rel="stylesheet" href="{{ STATIC_URL }}css/shield_study_10/main.css" type="text/css" />
+  </head>
+  <body>
+      <div class="container">
+        <div class="title-container">
+          <h1 class="title">Page Reload Study</h1>
+            <a href="https://addons.mozilla.org/firefox/downloads/latest/page-reload-study-1/?src=ext-shield-study-10"><button class="button primary large">Start Now</button></a>
+        </div>
+        <div class="content"><p>Mozilla wants to figure out the best way to detect when you are having trouble viewing a website. This will allow us to figure out why the website might be broken. Participating in this study will help us find out! </p>
+<h2 id="what-will-happen-next">What will happen next</h2>
+<p><strong>If you agree</strong> to participate:</p>
+<ol>
+<li>Continue using Firefox as you normally would. Depending on how you use the Web, you may not notice any changes.</li>
+<li>You might occasionally receive prompts that ask you about why you reload (refresh) a web page in Firefox.</li>
+<li>After a few weeks, the study will end on its own. Your Firefox will go back to normal.</li>
+<li>We will ask you to fill out a quick <strong>optional survey</strong> to tell us about your experience. The survey should take less than 5 minutes to complete.</li>
+</ol>
+<h2 id="leaving-the-study">Leaving the study</h2>
+<p>The study will expire on its own in a few weeks. You may leave the study early. To do so, follow these steps:</p>
+<ol>
+<li>Type about:addons into the location bar, and press enter.</li>
+<li>Find <strong>Page Reload Study 1</strong> in the Extensions list.</li>
+<li>Click the remove button.</li>
+<li>If you opt out of the study early we will ask you to fill out a survey. We’re interested in hearing about your experience, even if you didn’t participate in the entire study.</li>
+</ol>
+<p>Opting out of a study does not prevent you from participating in future studies.</p>
+<h2 id="your-privacy">Your privacy</h2>
+<h3 id="all-shield-studies">All Shield Studies</h3>
+<p>Every Shield Study collects data about important study events, such as install, uninstall, daily usage, and end of study. Studies also include an optional survey.</p>
+<p>This data is associated with <a href="https://gecko.readthedocs.io/en/latest/toolkit/components/telemetry/telemetry/index.html">Firefox Telemetry collection</a>.</p>
+<h3 id="this-particular-study">This particular study</h3>
+<p>In addition to the data collected by all Shield Studies, Mozilla will collect information about:</p>
+<ul>
+<li>High level domains and subdomains of websites you visit that may be broken</li>
+<li>Your interaction associated with reloading websites</li>
+</ul>
+<p>Mozilla will keep this information private; any data will only be shared in aggregate or anonymous form.</p>
+<p>You can learn more about the data collection for this study <a href="https://github.com/raymak/page-reload/wiki/Data-Collection">here</a>.</p>
```

Comment:
Can this link get a better description than `here` for accessibility reasons? Maybe something like `in our Wiki` or similar. We had this in pretty much all other shield studies and most recently it got fixed so feel free to get inspired there.